src/ownership-and-deconstruction.rst

```diff
@@ -258,65 +258,47 @@ Passing Conventions
 .. rubric:: Legality Rules
 
 :dp:`fls_fvwx2ufeyzcs`
-A :t:`passing convention` is a mechanism by which a :t:`value` is passed to and
-from a :t:`function`.
-
-:dp:`fls_1gyeqfpe7m1m`
-A :t:`value` is subject to a :t:`passing convention` when the :t:`value` is
-
-* :dp:`fls_jag0ud2lv08`
-  Assigned using an :t:`assignment expression`,
-
-* :dp:`fls_bkuz12srez4`
-  Bound to a :t:`function parameter` in a :t:`call expression`,
-
-* :dp:`fls_owltkb5i2lah`
-  Bound to a :t:`pattern`,
-
-* :dp:`fls_olqpm32j8va6`
-  Captured by a :t:`capture expression`,
-
-* :dp:`fls_czune894326w`
-  Returned from a :t:`function`.
-
-* :dp:`fls_qi148dixkp0w`
-  **More?**
+A :t:`passing convention` is the mechanism that defines how a :t:`value` is
+transferred between :t:`[place]s`. **place is not the appropriate term here**
```

Comment:
Is "place" still not the appropriate term?

Reply:
I am still unsure about the word, given `place expression` and the like

Reply:
Fair enough.